protoarray/src/test/java/tech/pegasys/teku/protoarray/VotesTest.java

```diff
@@ -62,7 +62,7 @@ void votesTest() {
     //          0
     //         /
     //        2
-    forkChoice.processBlock(ZERO, getHash(2), getHash(0), Bytes32.ZERO, ONE, ONE);
+    forkChoice.processBlock(ZERO, getHash(2), getHash(0), Bytes32.ZERO, ONE, ONE, Bytes32.ZERO);
```

Comment:
wondering if we wrap this so we can read the tests more easily
`processForkChoiceBlock(ZERO, 2, 0, Bytes32.ZERO, ONE, ONE, Bytes32.ZERO` etc...
would remove all the 'unsigned' and 'gethash' and encapsulate that in the fn...

Reply:
Had a go at this - it winds up shorter but definitely not easier to read:

```
    //  Left branch
    processBlock(forkChoice, unsigned(1), 1, 0, 0, 0);
    processBlock(forkChoice, unsigned(2), 3, 1, 1, 0);
    processBlock(forkChoice, unsigned(3), 5, 3, 1, 0);
    processBlock(forkChoice, unsigned(4), 7, 5, 1, 0);
    processBlock(forkChoice, unsigned(4), 9, 7, 2, 0);

    //  Right branch
    processBlock(forkChoice, unsigned(1), 2, 0, 0, 0);
    processBlock(forkChoice, unsigned(2), 4, 2, 0, 0);
    processBlock(forkChoice, unsigned(3), 6, 4, 0, 0);
    processBlock(forkChoice, unsigned(4), 8, 6, 1, 0);
    processBlock(forkChoice, unsigned(4), 10, 8, 2, 0);
```

I mean, it wasn't particularly good before either but losing track of what numbers are hashes and what are slots etc makes it even worse.

Reply:
yeh i dont like that either :) ok im convinced